docs/roadmap.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Roadmap"
+linkTitle: "Roadmap"
+weight: 50
+slug: roadmap 
+---
+
+The following is only a selection of some of the major features we plan to implement in the near future. To get a more complete overview of planned features and current work, see the issue trackers for the various repositories, for example, the [Cortex repo](https://github.com/cortexproject/cortex/issues).
+
+## Making a NoSQL store optional
+
+Currently Cortex depends on a NoSQL store (like Bigtable, DynamoDB, Cassandra) for its index storage. This adds a lot of complexity and cost to manage and we are currently working on making the NoSQL store optional by using an ObjectStore for index storage as well.
+
+## helm charts and other packaging
+
+We don't provide a set of helm charts and dashboards / alerts to our users yet. This is one of the most requested features and something we will tackle in the immediate future. We also plan on publishing debs, rpms along with guides on how to run Cortex on bare-metal.
```

Comment:
we have helm chart here: https://github.com/cortexproject/cortex-helm-chart so I'd make this more about being first class citizen / better quality / more supported.

Reply:
+1 to getting more involvement in the helm chart. Community is putting in PRs but I'm the only reviewer of late and my time lately has been very sporadic